### PR TITLE
7.0 updates

### PR DESCRIPTION
**Pull request by iCollin**
changes:
- address `##*Add link to AddCommand Section*` from joey
- address `TBD: Attach link for [Available Templates](#available-templates).` from joey
- address `##*Add Links to uploading graphics (android/ iOS) sections*` from joey
- fix note in `#### Handling Choices, ChoiceSets and Commands  `
- fix link for `OnPermissionsChange`
- fix link to `App Service Guidelines`
- fix link to `App Service Guidelines` / `RPC Passing`
- fix link to mobile api since it moved to rpc spec repo
- fix formatting of list of RPCs that can be send un-encrypted even with encryption_required

could not find new link for `Standard Interaction Phrases` broken link, all others were fixed.

### CODE REVIEW

File: docs/Best Practices/Audio Books/index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
+Your Audiobook app should contain such basic functionality as `Play`, `Pause`, and `Resume`. You can use build in controls for these features by calling the `subscribeButton` RPC. This will allow you to not only create buttons on the infotainment system but also be informed when the user presses hard buttons, which might be placed on the steering wheel. It is important to also add voice control capabilities for these functions. To add a voice command use the `AddCommand` RPC but omit the `menuParams` argument. This will create a voice command without creating an item in the menu.   
 
-##*Add link to AddCommand Section*
+More information about this RPC can be found on the [AddCommand page](https://smartdevicelink.com/en/docs/hmi/master/ui/addcommand/).
```
Comment (ShobhitAd) [inline docs/Best Practices/Audio Books/index.md:7]:
```suggestion
More information about this RPC can be found on the [UI.AddCommand](https://smartdevicelink.com/en/docs/hmi/master/ui/addcommand/) page.
```

File: docs/Best Practices/Audio Books/index.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
 #### Display Information
 Your app should display information like title, author, and potentially additional items like chapter number. It is encouraged to use the following layout of information on the screen:
```
Comment (ShobhitAd) [inline docs/Best Practices/Audio Books/index.md:10]: The `##*Add link to DisplayInformation Section*` line should also be changed to link to https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/

File: docs/Policies/App Policies/index.md [hunk 3]
```diff
@@ -184,11 +184,11 @@ This field defines the properties of the app services that this application is a
...
-* `handled_rpcs` - This field defines a list of permissions for RPCs that an app service could intercept via the [RPC passing](https://smartdevicelink.com/en/guides/core/app-service-provider-guide/#rpc-passing) feature. 
+* `handled_rpcs` - This field defines a list of permissions for RPCs that an app service could intercept via the [RPC passing](https://smartdevicelink.com/en/guides/core/feature-documentation/app-service-guidelines/#rpc-passing) feature. 
     * If the `handledRPCs` field in the app's `PublishAppService.appServiceManifest` includes a function ID that is not in this list, the message will be rejected. 
     * This field is mandatory, but can be empty.
     * Subfields:
-        * `function_id` - The [function ID](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml) of the potential handled RPC. Can be a function ID which was introduced after Core's local RPC spec version.
+        * `function_id` - The [function ID](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml) of the potential handled RPC. Can be a function ID which was introduced after Core's local RPC spec version.
```
Comment (ShobhitAd) [inline docs/Policies/App Policies/index.md:191]:
```suggestion
        * `function_id` - The [function ID](https://github.com/smartdevicelink/rpc_spec#functionid) of the potential handled RPC. Can be a function ID which was introduced after Core's local RPC spec version.
```

**Review by ShobhitAd — CHANGES_REQUESTED** [review]
We should also re-run the DocRetriever script to update the Protocol spec and RPC spec pages.

```
cd doc_retriever
python3 DocRetriever.py
```

File: docs/Getting Started/Registering an Application/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ If the implementation of Core supports hash resumption, any time an item that ca
 
 ### OnPermissionChange
 
-After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionchange) in the RPC spec.
+After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionschange) in the RPC spec.
```
Comment (jacobkeeler) [inline docs/Getting Started/Registering an Application/index.md:22]:
```suggestion
After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionsChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionschange) in the RPC spec.
```

Comment (iCollin) [reply docs/Getting Started/Registering an Application/index.md]: Changed all instances of `OnPermissionChange` in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/8ccd06238eec3909a54ce58ddc921b280110186d

File: docs/Policies/App Policies/index.md (hunk 3 above)
Comment (iCollin) [reply docs/Policies/App Policies/index.md]: good call, added in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/2d12ff2688a645cbbb6dcad8fe764e24ae77cb73

File: docs/Best Practices/Audio Books/index.md (hunk 2 above)
Comment (iCollin) [reply docs/Best Practices/Audio Books/index.md:10]: good catch, added in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/2d12ff2688a645cbbb6dcad8fe764e24ae77cb73

File: docs/Best Practices/Audio Books/index.md (hunk 1 above)
Comment (iCollin) [reply docs/Best Practices/Audio Books/index.md]: updated in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/2d12ff2688a645cbbb6dcad8fe764e24ae77cb73, also capitalized PerformInteraction below.

**Review by ShobhitAd — APPROVED** [review]

**Pull request merged**